### ISSUE TITLE
Era-appropriate startup sound as emulator loads

### ISSUE THREAD

**Issue by mihaip** [mihaip/infinite-mac#83]
May not be entirely possible because audio playback has a gesture requirement.

**Comment by HHOhlers** [comment]
how about the website (HTML)  itself plays the sound? (not the emulator)

**Comment by that-ben** [comment]
No, you do not understand.  On most modern web browsers, you need to tap/click an element for audio playback to unlock.

**Comment by HHOhlers** [comment]
I do understand. To further your knowledge on the subject , you can read this article: https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide

**Comment by eduo** [comment]
I opened this issue and was half-expecting that this means having a floppy disk drive insert sound and read/write sound added. 

Virtual PC (I think, or softwindows or realpc) had this in early versions and it was so whimsical and memorable. It would sound like a floppy or a slow hard drive were spinning up to read the BIOS.

All 68K could have a floppy disk reading sound played, as a fun tidbit :D

**Comment by egnor** [comment]
...don't your emulators currently play startup sounds, if you've seeded the page with a click?

**Comment by mihaip** [comment]
I don't do any explicit startup sound playback, but some of the emulators that I added since I filed this issue (DingusPPC, Snow) do a more faithful emulation, which includes the startup sound playback from the ROM. 

To actually hear the sound, it does require a user gesture first, so it only works if navigating to the homepage first and then clicking "Run…" on an instance. That's the other change since the issue was filed - back then I only had https://system7.app and https://macos8.app, and they would directly start the machine, with no chance for a gesture.

Declaring this done.
